assets/css/styleguide.scss

```diff
@@ -442,24 +444,28 @@ $site-top:           124px;
   }
 }
 
-.usa-grid-example-blank {
-  text-align: center;
+.grid-example-blank {
+  .usa-grid {
+    text-align: center;
+  }
 }
 
-.usa-grid-text {
-  background: $color-grid-dark;
+.grid-text {
+  .usa-grid {
+    background: $color-grid-dark;
 
-  > * {
-  background: $color-grid-light;
-  color: #000;
-  }
+    > * {
```

Comment:
@msecret I just checked.. it's indented with the background property which is one level in from `.usa-grid` so I think that's correct:

<img width="453" alt="screen shot 2015-12-18 at 10 55 48 am" src="https://cloud.githubusercontent.com/assets/5249443/11904615/0f145d2c-a576-11e5-9dd0-5096264349ce.png">